src/operations/common/allReposInTeamRepoFinder.ts

```diff
@@ -15,9 +15,6 @@ const PageSize = 100;
 /**
  * Use a GraphQL query to find all repos for the current team,
  * or look locally if appropriate, in current working directory
- *
- * DEPRECATED: there's a better one in @atomist/sdm
- *
```

Comment:
wait why is this deprecation removed? This one is actually dangerous, because it works off the current directory, which does not fit the SDM's way of working, and people could use this accidentally and get very confused. How about we remove this one instead?

Reply:
It is being used in a lot of places. This for finding repos in graphql, not on the local file system.

Reply:
I pushed a commit. Tests don't run locally for me on this project